airbyte/_future_cdk/sql_processor.py

```diff
@@ -303,8 +303,8 @@ def _get_table_by_name(
     def _ensure_schema_exists(
         self,
     ) -> None:
-        """Return a new (unique) temporary table name."""
-        schema_name = self.sql_config.schema_name
+        dialect = self.get_sql_engine().dialect
+        schema_name = dialect.normalize_name(self.sql_config.schema_name)
         if schema_name in self._get_schemas_list():
             return
```

Comment:
I think we should try to use the internal normalizer, just to avoid depending too much on SQLAlchemy dialects. Our normalizer class has methods for both `normalize()` and `normalize_set(Iterable[str])`.

Something like `schema_name = self.normalizer.normalize(...)`...

Or perhaps just making `if schema_name in self._get_schemas_list()` into a case-insensitive check, something like `if self.normalizer.normalizer(schema_name) in self.normalizer.normalize_set(self._get_schemas_list()`?

Wdyt?